src/main/scala/org/broadinstitute/dsde/firecloud/model/ElasticSearch.scala

```diff
@@ -56,19 +56,18 @@ case class LibrarySearchResponse(
 
 // classes to create the ES queries in json format
 // {"query":{"match_all":{}}}"
-// {"query":{"wildcard":{"_all":"%s"}}}"
+// {"query":{"match":{"_all":"<str>"}}}"
 trait QueryMap
```

Comment:
Do you want to make this a sealed trait? I think if you do that, then when you use it in case statements, the compiler (and IDE) can give you warnings about whether you're handling all of them or not.